Missions/zagor_zagorsk_biglakes/mission.sqm

```diff
@@ -0,0 +1,815 @@
+			id=89;
+		};
+		class Item32
+		{
+			dataType="Marker";
+			position[]={-1919.0807,194.92407,2902.8101};
+			name="A3E_ExtractionSpawnPos1";
+			type="Empty";
+			angle=296.77734;
+			id=90;
+			atlOffset=-9.0059204;
+		};
+		class Item33
+		{
+			dataType="Marker";
+			position[]={20395.938,185.04565,12363.503};
+			name="Respawn_east";
+			type="Empty";
+			colorName="ColorRed";
+			id=91;
+			atlOffset=-43.085953;
+		};
+		class Item34
+		{
+			dataType="Marker";
+			position[]={20.040699,261.28027,12941.691};
+			name="Respawn_west";
+			type="Empty";
+			colorName="ColorBlue";
+			id=92;
+			atlOffset=-2.4031372;
+		};
+		class Item35
+		{
+			dataType="Marker";
+			position[]={31.4844,249.83032,12920.052};
+			name="Boot_Hill";
+			type="Empty";
+			id=93;
+			atlOffset=-12.930084;
+		};
+		class Item36
+		{
+			dataType="Marker";
+			position[]={8400.3447,323.24854,2470.3726};
+			name="RoadBlockNullRoad";
+			type="Empty";
+			colorName="ColorRed";
+			id=94;
+			atlOffset=81.147705;
+		};
+		class Item37
+		{
+			dataType="Marker";
+			position[]={9563.4014,133.94966,20463.588};
+			name="TrafficMarker_North";
+			type="Empty";
+			id=95;
+		};
+		class Item38
+		{
+			dataType="Marker";
+			position[]={20471.457,88.528564,9493.6895};
+			name="TrafficMarker_East";
+			type="Empty";
+			id=96;
+			atlOffset=-124.41547;
+		};
+		class Item39
+		{
+			dataType="Marker";
+			position[]={8600.9092,119.24927,11.169556};
+			name="TrafficMarker_South";
+			type="Empty";
+			id=97;
+			atlOffset=-134.12389;
+		};
+		class Item40
+		{
+			dataType="Marker";
+			position[]={10.158245,253.21538,7796.1924};
+			name="TrafficMarker_West";
+			type="Empty";
+			id=98;
+		};
+		class Item41
+		{
+			dataType="Marker";
+			position[]={17918.627,231.91257,6787.8428};
+			name="TrafficMarker_AirField";
+			type="Empty";
+			id=99;
+			atlOffset=9.4650726;
+		};
+		class Item42
+		{
+			dataType="Logic";
+			class PositionInfo
+			{
+				position[]={10238.634,237.50002,10224.596};
+				angles[]={6.2812018,0,0.035985984};
+			};
+			name="server";
+			id=100;
+			type="Logic";
+		};
+		class Item43
+		{
+			dataType="Logic";
+			class PositionInfo
+			{
+				position[]={10233.815,237.22311,10239.054};
+				angles[]={0.010000871,0,0.055940546};
+			};
+			name="center";
+			id=101;
+			type="Logic";
+		};
+		class Item44
+		{
+			dataType="Logic";
+			class PositionInfo
+			{
+				position[]={210.73892,334.57886,185.94971};
+			};
+			name="SouthWest";
+			description="limits used area of the map, together with NorthEast logic";
+			id=102;
+			type="Logic";
+			atlOffset=100.65431;
+		};
+		class Item45
+		{
+			dataType="Logic";
+			class PositionInfo
+			{
+				position[]={20240.07,185.56166,20282.438};
+				angles[]={6.2711854,0,0.033987202};
+			};
+			name="NorthEast";
+			description="limits used area of the map, together with SouthWest logic";
+			id=103;
+			type="Logic";
+		};
+		class Item46
+		{
+			dataType="Group";
+			side="East";
```

Comment:
Must replace `East` with `{* PLAYERSIDE *}` here and in player configs, player `init` fields with `init="{* PLAYER_INIT *}";`, player types with `type="{* PLAYERUNIT_1 *}";` (1,2,...,10). You can see example in Abramia mission config. Right now "vs East" missions are not playable.

Reply:
Fixed